### PR TITLE
address CVE-2022-39227

### PR DESCRIPTION
**Pull request by Redent0r**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
Address CVE-2022-39227

###### Change Log  

- CVE-2022-39227

###### Does this affect the toolchain?  

**NO**

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-2022-39227

### CODE REVIEW

File: SPECS/python-jwt/python-jwt.spec [hunk 1]
```diff
@@ -1,3 +1,4 @@
+# This package refers to PyJWT(https://github.com/jpadilla/pyjwt). Not to be confused with python-jwt(https://github.com/davedoesdev/python-jwt)
```
Comment (nicogbg) [inline SPECS/python-jwt/python-jwt.spec:1]: you need to increment release number because spec has changed (even if it is only a comment)

**Review by nicogbg — APPROVED** [review]

File: SPECS/python-jwt/python-jwt.spec (hunk 1 above)
Comment (Redent0r) [reply SPECS/python-jwt/python-jwt.spec:1]: Updated. Thanks!

**Review by mfrw — COMMENTED** [review]
I would encourage to have a pointer of the package before we merge.
e.g
`python-jwt: address CVE-2022-39227`

This would make it a little easier to parse latter on with a `git log --oneline` for changelog generation :)

**Comment by Redent0r** [comment]
@mfrw you mean in the commit message right? added

**Review by jslobodzian — APPROVED** [review]

**Pull request merged**